modules/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer.js

```diff
@@ -154,9 +155,34 @@ export default class GPUScreenGridLayer extends Layer {
     });
   }
 
+  getPickingInfo({info, mode}) {
+    const {index} = info;
+    if (index >= 0) {
+      let {aggregationData} = this.state;
+      if (!aggregationData) {
+        aggregationData = {
+          countsData: this.state.countsBuffer.getData(),
```

Comment:
On a higher level, could the buffers be hidden inside the GPU aggregator, rather than the layer having to know about them?